ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-release-4.18.yaml

```diff
@@ -276,7 +276,7 @@ tests:
     workflow: openshift-qe-cluster-density-v2
   timeout: 5h0m0s
 - always_run: false
-  as: qe-perfscale-aws-ovn-small-cluster-density
+  as: ovncore-perfscale-aws-ovn-small-cluster-density
```

Comment:
This is not a core-ovn job, is it?

Reply:
Based on the conversation, adjusted cluster_profiles for the below:

```
24 Nodes
perfscale-aws-ovn-small-cluster-density-v2
perfscale-aws-ovn-small-node-density-cni
====================
120 Nodes
perfscale-aws-ovn-medium-cluster-density-v2
perfscale-aws-ovn-large-node-density-cni
====================
501 Nodes
ovncore-perfscale-aws-ovn-large-cluster-density-v2
ovncore-perfscale-aws-ovn-large-node-density-cni
====================
750 Nodes
ovncore-perfscale-aws-ovn-xlarge-cluster-density-v2
ovncore-perfscale-aws-ovn-xlarge-node-density-cni
```